src/components/radio/radio.ts

```diff
@@ -53,9 +71,13 @@ export class MdRadioGroup implements AfterContentInit {
   /** The currently selected radio button. Should match value. */
   private _selected: MdRadioButton = null;
 
+  private _changeSubscription: {unsubscribe: () => any} = null;
+
+  onTouched: () => any = () => {};
+
   /** Event emitted when the group value changes. */
   @Output()
-  change: EventEmitter<MdRadioChange> = new EventEmitter();
+  change: EventEmitter<any> = new EventEmitter();
```

Comment:
We should keep the event object (checkbox should also have one eventually), and make it so that you just call the `fn` argument with the value extracted from the event object:
```ts
registerOnChange(fn: any) {
  if (this._changeSubscription) {
    this._changeSubscription.unsubscribe();
  }
  this._changeSubscription = <{unsubscribe: () => any}>this.change.subscribe(changeEvent => {
    fn(changeEvent.value);
  });
}```